dist/js/bootstrap-switch.js

```diff
@@ -7,13 +7,57 @@
   * @license Apache-2.0
   */
 
-'use strict';
+(function (global, factory) {
+  if (typeof define === "function" && define.amd) {
+    define(['exports', 'jQuery'], factory);
+  } else if (typeof exports !== "undefined") {
+    factory(exports, require('jQuery'));
+  } else {
+    var mod = {
+      exports: {}
+    };
+    factory(mod.exports, global.jQuery);
+    global.bootstrapSwitch = mod.exports;
+  }
+})(this, function (exports, _jQuery) {
+  'use strict';
+
+  Object.defineProperty(exports, "__esModule", {
+    value: true
+  });
+
+  var _jQuery2 = _interopRequireDefault(_jQuery);
+
+  function _interopRequireDefault(obj) {
+    return obj && obj.__esModule ? obj : {
+      default: obj
+    };
+  }
+
+  function _classCallCheck(instance, Constructor) {
+    if (!(instance instanceof Constructor)) {
+      throw new TypeError("Cannot call a class as a function");
+    }
+  }
 
-var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
+  var _createClass = function () {
```

Comment:
it's odd that this has changed. This and the css changes makes me think that your build config is different to ours. I think it might be best to keep dist out of sync until a release is cut.